.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -42,7 +42,8 @@ This role should rotate between LTS releases
 
 - [ ] Backport changes, create a local branch in jenkinsci/jenkins, run the [list-issue-commits script](https://github.com/jenkins-infra/backend-commit-history-parser/blob/master/bin/list-issue-commits) to locate commits via jira ID, some manual work is required to locate them if the issue ID wasn't present at merge time, backport with `git cherry-pick -x $commit`
 
-- [ ] Open backporting PR with into-lts label and summary of changes in description from [lts-candidate-stats script](https://github.com/jenkins-infra/backend-commit-history-parser/blob/master/bin/lts-candidate-stats) and the selected [Jira lts-candidates](https://issues.jenkins-ci.org/issues/?filter=12146)
+- [ ] Open backporting PR with into-lts label and summary of changes in description from [lts-candidate-stats script](https://github.com/jenkins-infra/backend-commit-history-parser/blob/master/bin/lts-candidate-stats) and the selected [Jira lts-candidates](https://issues.jenkins-ci.org/issues/?filter=12146)  
+Also check the [release repository](https://github.com/jenkins-infra/release/issues?q=is%3Aclosed+label%3Alts-candidate+) for additional candidates.
```

Comment:
And packaging

Reply:
Oic, if packaging uses GH issues, it'd make sense to archive the Jira, no?

I've added the packaging repo, just need to create the label, I have no permission there.